syntax = "proto3";



message ParameterMessage {
    bool bool_param = 1;
    int32 int_param = 2;
    double float_param = 3;
    string string_param = 4;
}
